hicommonwealth/commonwealth
# [Admin-Onboarding] New Community Integrations Screen

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6025

## Description of Changes
- Added new community integrations screen with Community Directory, Snapshot, Discord (untested), Webhooks, Custom TOS, and Url integrations
- Added mobile responsiveness

## "How We Fixed It"
N/A

## Test Plan
- Enable the `FLAG_NEW_ADMIN_ONBOARDING` flag
- Go to any community where you are an admin
- Go to the `manage/integrations` page
- Verify that the page is responsive on different screen sizes
- Verify that Community Directory integration is working as expected
- Verify that Snapshot integration is working as expected
- Verify that Discord integration is working as expected
- Verify that Webhooks integration is working as expected
- Verify that Custom TOS is working as expected
- Verify that Custom Url "contact us" link opens the device mailbox.
- CA and verify there aren't any issues.

## Deployment Plan
N/A

## Other Considerations
I wasn't able to test discord integration on local properly, because it required some API token's to configure.
